cloudlift/config/service_configuration.py

```diff
@@ -229,9 +238,80 @@ def _validate_changes(self, configuration):
                 },
                 "logging": {
                     "oneOf": [
-                        {"type": "string", "pattern": "^(awslogs|fluentd|null)$"},
+                        {"type": "string", "pattern": "^(awslogs|awsfirelens|null)$"},
```

Comment:
Lets keep default awslogs untill we have full release.  This is avoid  issues during prod deployments. @shoan what you think ?

Reply:
We should keep awslogs as default.